sql/core/benchmarks/DataSourceReadBenchmark-results.txt

```diff
@@ -2,268 +2,268 @@
+SQL Json                                      6167 / 6180          2.6         392.1       2.8X
+SQL Parquet Vectorized                         134 /  142        117.5           8.5     128.2X
+SQL Parquet MR                                1659 / 1740          9.5         105.5      10.3X
+SQL ORC Vectorized                             225 /  229         69.9          14.3      76.3X
+SQL ORC Vectorized with copy                   231 /  235         68.2          14.7      74.4X
+SQL ORC MR                                    1287 / 1388         12.2          81.8      13.3X
+
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 on Mac OS X 10.13.6
+Intel(R) Core(TM) i7-6920HQ CPU @ 2.90GHz
 Parquet Reader Single INT Column Scan:   Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------
-ParquetReader Vectorized                       257 /  274         61.3          16.3       1.0X
-ParquetReader Vectorized -> Row                259 /  264         60.8          16.4       1.0X
+ParquetReader Vectorized                       178 /  187         88.2          11.3       1.0X
+ParquetReader Vectorized -> Row                174 /  184         90.3          11.1       1.0X
 
-OpenJDK 64-Bit Server VM 1.8.0_181-b13 on Linux 3.10.0-862.3.2.el7.x86_64
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_131-b11 on Mac OS X 10.13.6
+Intel(R) Core(TM) i7-6920HQ CPU @ 2.90GHz
 SQL Single BIGINT Column Scan:           Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------
-SQL CSV                                     32537 / 32554          0.5        2068.7       1.0X
-SQL Json                                    12610 / 12668          1.2         801.7       2.6X
-SQL Parquet Vectorized                         258 /  276         61.0          16.4     126.2X
-SQL Parquet MR                                2422 / 2435          6.5         154.0      13.4X
-SQL ORC Vectorized                             378 /  385         41.6          24.0      86.2X
-SQL ORC Vectorized with copy                   381 /  389         41.3          24.2      85.4X
-SQL ORC MR                                    1797 / 1819          8.8         114.3      18.1X
-
-OpenJDK 64-Bit Server VM 1.8.0_181-b13 on Linux 3.10.0-862.3.2.el7.x86_64
-Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
+SQL CSV                                     21253 / 21542          0.7        1351.2       1.0X
+SQL Json                                      8208 / 8209          1.9         521.9       2.6X
+SQL Parquet Vectorized                         180 /  241         87.3          11.5     117.9X
+SQL Parquet MR                                1769 / 1801          8.9         112.5      12.0X
+SQL ORC Vectorized                            3271 / 3277          4.8         207.9       6.5X
```

Comment:
@gengliangwang . Just a surprise, `86.2X -> 6.5X`?

Reply:
Could you regenerate the result again?
I think this might be your Macbook issue at this moment of time. 
Please see line 71. It's faster that this. (Usually, line 71 is slower than this due to copying).

Reply:
OK, let me re-run it.
Thanks for the check!

Reply:
Done in https://github.com/apache/spark/pull/22965/commits/3067a6d1f63c93b4295425d90e5894d27c840995 .